### ISSUE TITLE
Accessibility: MAS42A: High Contrast: The logos are not visible  for the links in High contrast white Theme

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#16]
#A11yMAS
#MAS42A
#Wipro
#vscode-website
#FireFox

**Prerequisite:**
**_1. Change system setting to High contrast White_**
    Go to Settings-->Ease of Access-->High Contrast.
    Select "High Contrast White" from the "Choose a theme" dropdown. Click on Apply.

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0
Firefox Version:50.1.0

**Repro Steps:**
**_Scenario-1:_**
1. Open [https://code.visualstudio.com/](https://code.visualstudio.com/) in edge.
2. Naviagte to the "Follow@code" and "Microsoft" link at  the footer of the page.
3. Observe the logos for the links in high contrast white.

Note: Same issue is observed for the below links:
[https://code.visualstudio.com/docs
](https://code.visualstudio.com/docs
)[http://code.visualstudio.com/Download
](http://code.visualstudio.com/Download
)http://code.visualstudio.com/insiders
https://code.visualstudio.com/blogs
https://code.visualstudio.com/updates
https://code.visualstudio.com/docs/supporting/faq
https://code.visualstudio.com/docs/editor/codebasics
https://code.visualstudio.com/docs/introvideos/overview
https://code.visualstudio.com/docs/editor/accessibility
https://code.visualstudio.com/docs/editor/codebasics
https://code.visualstudio.com/docs/customization/userandworkspace
https://code.visualstudio.com/docs/languages/overview
https://code.visualstudio.com/docs/setup/setup-overview
https://code.visualstudio.com/docs/runtimes/nodejs

**_Scenario-2:_**
1.Open http://code.visualstudio.com/docs/editor/codebasics in edge.
2.Navigate to the "IN THIS ARTICLE" at RHS pane.
3.Observe the logo for the "Tweet" link below the "IN THIS ARTICLE"

**Actual Result:**
**_Scenario-1:_**
The logos are missing for the links "Follow@code" and "Microsoft" at the footer of the page in high contrast white.

**_Scenario-2:_**
The logo is missing for "Tweet"  link above the footer of the page in high contrast white

**Expected Result:**
**_Scenario-1:_**
The logos should be visible for the "Follow@code" and "Microsoft" links at footer of the page in high contrast white.

**_Scenario-2:_**
The logo should be  visible for "Tweet"  link above the footer of the page in high contrast white

**User Impact:**
The persons with colour blindness and low vision may miss the logos for the links.

**Screenshots:**
[Scenario1_logos are not visible in HCW.zip](https://github.com/wiproaccess/vscode-website/files/687115/Scenario1_logos.are.not.visible.in.HCW.zip)
[Scenario2_Tweet logo not visible in HCW.zip](https://github.com/wiproaccess/vscode-website/files/687116/Scenario2_Tweet.logo.not.visible.in.HCW.zip)

**Comment by waderyan** [comment]
![image](https://cloud.githubusercontent.com/assets/2449568/26736788/6a98b0b8-477c-11e7-9dfb-ae4700c59030.png)

![image](https://cloud.githubusercontent.com/assets/2449568/26736801/72e4ba82-477c-11e7-93a1-cf3f2a1b3a28.png)

**Comment by waderyan** [comment]
Fix for Microsoft logo - ab223454eff0806c2f1d069dea4fa3124df4ddfa

**Comment by chrisdias** [comment]
The twitter icon not showing up in HC mode is something outside of our control and we believe there is sufficient information presented (both in HC and through narrator) that it will be clear as to what the button does.  We will submit a bug to twitter for them to fix their button.  

Given that, we think this issue is fully resolved.

**Comment by chrisdias** [comment]
https://twitter.com/chrisdias/status/872520701986521088
